### PR TITLE
Vet Clinic: - Feature/update table

### PR DESCRIPTION
**Pull request by Bennyjoez**
## Vet Clinic: - Feature/update table 🪶 

I have managed to effect the following changes in my database: ♻️ 👨🏾‍💻 
 - I have added an extra column to the animals' table. ➕ 
 - I have Added species for different animals. ➕ 
 - I have queried the number of animals. ➕ 
 - I have worked with transactions in the database. ➕ 
 - I have queried the data with GROUP BY clauses and ORDER BY clauses. ➕ 
 - I have taken screenshots of the results. ➕ 
Screenshot from 2023-06-13 17-15-21
Screenshot from 2023-06-13 17-20-31
Screenshot from 2023-06-13 17-35-35
Screenshot from 2023-06-13 17-35-42
Screenshot from 2023-06-13 18-36-44

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -8,3 +8,13 @@ VALUES
...
+INSERT INTO animals 
+  (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-08-02', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, true, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, true, 45),
+  ('Boarmon', '2005-06-07', 7, true, 20.4),
+  ('Blossom', '1998-08-13', 3, true, 17),
+  ('Ditto', '2022-05-14', 4, true, 22);
```
Comment (mukhammadrizooff) [inline data.sql]: - [ ] [REQUIRED] Good job so far! Please give correct `data` to your `VALUES` =>

- `[YOURS WRONG VALUES]`
```sql
INSERT INTO animals 
  (date_of_birth, weight_kg)
VALUES
  ('2020-08-02', 11),
  ('2021-11-15', 5.7),
  ('1993-04-02', 12.13),
  ('2005-06-12', 45),
  ('1998-08-13', 17),
```

- `[CORRECT VALUES]`
```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Blossom', '1998/10/13', '3', 'true', '17.0');
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Bennyjoez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Bennyjoez) [reply data.sql]: Hello @mukhammadrizooff. 
Thank you for pointing the issue out. I was able to identify it and write a script to fix the data. 📜  Issue solved 🟢  
Screenshot from 2023-06-14 05-59-55

File: data.sql [hunk 2]
```diff
@@ -8,3 +8,20 @@ VALUES
...
+
+-- CORRECT DATA
+BEGIN;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE name IN ('Charmander', 'Plantmon', 'Squirtle', 'Angemon');
+COMMIT;
```
Comment (omar-muhamad) [inline data.sql]: - [x] As required by the previous reviewer, you suppose to correct the entered data values, not make a transaction to correct them, Please fix that.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Bennyjoez,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Bennyjoez,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Bennyjoez** [comment]
Thank you very much, gentlemen. 👍🏾  @mukhammadrizooff  @Omar-Muhamad

**Pull request merged**